frontend/src/app/accountCreation/LoadingCard/LoadingCard.tsx

```diff
@@ -0,0 +1,19 @@
+import { Card } from "../../commonComponents/Card/Card";
+import { CardBackground } from "../../commonComponents/CardBackground/CardBackground";
+import iconLoader from "../../../../node_modules/uswds/dist/img/loader.svg";
+
+interface Props {
+  message: string;
+}
+
+export const LoadingCard = (props: Props) => {
```

Comment:
This component is used as a transition between pages, it gets rendered when `loading` is true on most components